src/commands/delifeq.json

```diff
@@ -7,6 +7,7 @@
         "arity": 3,
         "function": "delifeqCommand",
         "command_flags": [
+            "FAST",
```

Comment:
i am not sure if we need the ACCESS flag, i dont know...
```
                "flags": [
                    "RM",
                    "ACCESS",
                    "DELETE"
                ],
```

Reply:
I think RM, ACCESS and DELETE are keyspec flags. Not command flags. I checked other commands.

DEL: command flag WRITE, keyspec flags RM, DELETE.

Reply:
Even the `SET` command has `ACCESS`, so I think DELIFEQ should have it too.

Reply:
:thinking: DELIFEQ uses it for the condition to delete or not. Does it mean access?

I think not. Compare to SETNX. It uses the it as a condition too. It does not have the ACCESS flag.

Reply:
ok, removed.